ansible/inventory/ec2.py

```diff
@@ -0,0 +1,1705 @@
+#!/usr/bin/env python
+
+'''
+File Source
+=================================
+https://raw.githubusercontent.com/ansible/ansible/v2.5.0b1/contrib/inventory/ec2.py
```

Comment:
I see [2.5.0 is tagged](https://github.com/ansible/ansible/blob/v2.5.0/contrib/inventory/ec2.py). Still 2.5.0b1?
And when is this copied file planned for removal? When repo moved to 2.5.0? Can it be moved already? So not to add this 1.7k lines file to repo?

Reply:
it's not yet avaliable as ansible plugin